### ISSUE TITLE
TRNSYS type not found

### ISSUE THREAD

**Issue by Gim07** [SPF-OST/pytrnsys_gui#441]
Hi, I have an issue when i try to run the mass flow solver in the example solar_dhw_GUI. A pop-up windows with a TRNSYS error display that there are 2 errors in the deck file. I have already encountered these errors, and i suppose that the problem is related to the .dll not found by TRNSYS (TRNSYS Error 105). I had copy the dll in \pytrnsys_data\ddcks\dlls in to the C:\Trnsys17\UserLib\ReleaseDLLs directory as reported in the installation guide, but it still doesn't work.  
Can you help me to resolve this problem?

**Comment by zuckerruebe** [comment]
Hi Giacomo. Could you attach the `solar_dhw_GUI.log` file in the project folder that is generated when the mass flow solver is run?

**Comment by Gim07** [comment]
Yes of course. I tried also other examples, and custom project created following the guide, but I encountered always the same issue.
[solar_dhw_GUI_mfs.log](https://github.com/SPF-OST/pytrnsys_gui/files/11271205/solar_dhw_GUI_mfs.log)

**Comment by zuckerruebe** [comment]
It seems TRNSYS cannot find the implementations/DLLs for types 150 and 935:

![grafik](https://user-images.githubusercontent.com/11159050/233348498-35e14a37-f362-453d-a59a-31812fc84e96.png)

The DLL for type 931 is there, but for some reason TRNSYS cannot load it. We're looking into and will get back to you. I hope that's ok?
